src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/QualityGuidelines/MarkMembersAsStaticTests.cs

```diff
@@ -600,6 +600,84 @@ End Namespace
             }.RunAsync();
         }
 
+        [Fact]
+        [WorkItem(4995, "https://github.com/dotnet/roslyn-analyzers/issues/4995")]
+        [WorkItem(5110, "https://github.com/dotnet/roslyn-analyzers/issues/5110")]
+        public async Task AttributeImplementingNUnitITestBuilder_NoDiagnostic()
+        {
+            var referenceAssemblies = AdditionalMetadataReferences.DefaultWithNUnit;
+
+            await new VerifyCS.Test
+            {
+                ReferenceAssemblies = referenceAssemblies,
+                TestState =
+                {
+                    Sources =
+                    {
+                        @"
+public class Test
+{{
+    [CustomNUnit.MyTestBuilder]
+    public void Method1() {{}}
+}}
```

Comment:
```suggestion
{
    [CustomNUnit.MyTestBuilder]
    public void Method1() {}
}
```

Reply:
Thanks @Youssef1313. Sorry I made the change too quickly... 🤦‍♂️